koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -424,6 +425,16 @@ def _write_logs(self, build_id, logs_dir, platforms: list = None):
                 task_platform = 'noarch'
 
             if task_platform not in logfiles:
+                if task_platform != 'noarch' and not final_platforms:
+                    final_platforms = self.osbs().get_final_platforms(build_id)
+
+                    if not final_platforms:
+                        self.logger.info("Couldn't obtain final platforms from build")
+                        final_platforms = platforms
+
+                if (task_platform != 'noarch') and (task_platform not in final_platforms):
+                    continue
```

Comment:
In atomic-reactor, logs for disabled platforms are written to osbs-build.log. Do we want to do the same thing here, or just skip them?

Reply:
you mean that we are logging execution of all build tasks in osbs-build.log?

well we don't have to do it here at all, as those shouldn't be in logs in koji

Reply:
only the disabled ones, so there is something like `<platform> is not enabled for this build, exiting`

But yeah, we can probably just skip those here